src/CMakeLists.txt

```diff
@@ -920,6 +920,7 @@ if(WITH_TESTS AND WITH_DMCLOCK_TESTS)
   add_subdirectory(dmclock/support/test)
 endif(WITH_TESTS AND WITH_DMCLOCK_TESTS)
 
+add_custom_target(crypto_plugins)
```

Comment:
we could restructure this later on by moving this line to src/crypto/CMakeLists.txt, and move the following if block to that file .